bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -74,7 +74,7 @@ export default withRouter(injectIntl(withModalMounter(createContainer((
   // Check if user is kicked out of the session
   Users.find({ userId: Auth.userID }).observeChanges({
     changed(id, fields) {
-      if (fields.user && fields.user.kicked) {
+      if (fields.ejected && fields.connectionStatus === 'offline') {
```

Comment:
Why the check for the connection status? In which case a ejected user is not offline?